update ui for account detail page

JustinBeBoy opened the pull request:
resolve #342

raedah commented:
I havent built the PR, but the screenshot design looks okay.

ademuanthony commented on web/routes/handlers.go:
```diff
@@ -6,14 +6,16 @@ import (
 	"net/http"
 	"strconv"
 
```
remove this empty line and rearrange import

ademuanthony commented on web/static/app/src/controllers/accounts_controller.js:
```diff
@@ -0,0 +1,55 @@
+
+      const postData = (accountElement.checked) ? `hide-account=${accountNumber}`: `reveal-account=${accountNumber}`
+      axios.put('/settings', postData).then((response) => {
+        let result = response.data 
```
`const result = response.data`

ademuanthony commented on web/static/app/src/controllers/accounts_controller.js:
```diff
@@ -0,0 +1,55 @@
+    }
+
+    updateDefaultAccount (e) {
+       let defaultAccountEl = e.currentTarget
```
`const`

ademuanthony commented on web/static/app/src/controllers/accounts_controller.js:
```diff
@@ -0,0 +1,55 @@
+
+    updateDefaultAccount (e) {
+       let defaultAccountEl = e.currentTarget
+       let defaultAccount = defaultAccountEl.getAttribute("data-account")
```
`const`

ademuanthony commented on web/static/app/src/controllers/accounts_controller.js:
```diff
@@ -0,0 +1,55 @@
+       // uncheck all other accounts that were previously marked default
+       this.defaultAccountTargets.forEach((el, i) => {
+           if (el.checked && el.getAttribute("data-account") != defaultAccount) {
+               el.checked = !el.checked
```
`el.checked = false` i think since we are not trying to toggle the checked state

ademuanthony commented on web/static/app/src/controllers/accounts_controller.js:
```diff
@@ -0,0 +1,55 @@
+       // post data 
+       const postData = `default-account=${defaultAccount}`
+       axios.put('/settings', postData).then((response) => {
+            let result = response.data 
```
`const`

ademuanthony commented on web/views/accounts.html:
```diff
@@ -1,15 +1,101 @@
+                                        {{ end }}
+                                    {{ end }}
+
+                                    <input data-target="accounts.hideAccount" data-action="change->accounts#toggleHideAccount"
```
there is a need to style this checkbox and the label so that the check mark will stay on the left

codemaestro64 commented:
Updated screenshot 

![Screenshot from 2019-05-06 23-23-49](https://user-images.githubusercontent.com/42093751/57259295-f582d000-7056-11e9-8d58-1f9aa12f4314.png)

UMichael commented:
This builds and gives the graphical output above

Sirmorrison commented:
![image](https://user-images.githubusercontent.com/27733432/57474808-e4220980-728a-11e9-98c9-93352769d4a3.png)

when user wallet is 1 there is this line at the bottom. doest look good on the ui. can that bottom border be removed?

codemaestro64 commented:
> ![image](https://user-images.githubusercontent.com/27733432/57474808-e4220980-728a-11e9-98c9-93352769d4a3.png)
> 
> when user wallet is 1 there is this line at the bottom. doest look good on the ui. can that bottom border be removed?

fixed. 
![Screenshot from 2019-05-10 15-40-35](https://user-images.githubusercontent.com/42093751/57535378-045cd180-733a-11e9-8fe4-f4678fa5387e.png)

Sirmorrison commented:
![image](https://user-images.githubusercontent.com/27733432/57565980-dcfc1800-73be-11e9-8b3b-52bef52b0455.png)

we are almost there could the padding at the top and bottom be removed or reduced. The space is way too much and makes the ui look .....

codemaestro64 commented:
> ![image](https://user-images.githubusercontent.com/27733432/57565980-dcfc1800-73be-11e9-8b3b-52bef52b0455.png)
> 
> we are almost there could the padding at the top and bottom be removed or reduced. The space is way too much and makes the ui look .....

fixed

codemaestro64 commented:
Updated screenshot 

![Screenshot from 2019-05-11 09-24-02](https://user-images.githubusercontent.com/42093751/57567280-99110f00-73ce-11e9-8f10-8ac6fcc62f64.png)

Sirmorrison commented:
> 
> 
> Updated screenshot
> 
> ![Screenshot from 2019-05-11 09-24-02](https://user-images.githubusercontent.com/42093751/57567280-99110f00-73ce-11e9-8f10-8ac6fcc62f64.png)

remove the bottom padding

Sirmorrison commented:
Tested and works fine

JustinBeBoy committed:
update ui for account detail page

- add inset left for SubPageLayout and SubpageHeader in page.go
- add inset and recreate function in account detail page

oshorefueled commented on ui/account_details_page.go:
```diff
@@ -249,13 +227,34 @@ func (pg *acctDetailsPage) accountInfoLayout(gtx layout.Context) layout.Dimensio
+		layout.Rigid(func(gtx C) D {
+			return layout.Flex{Axis: layout.Horizontal}.Layout(gtx,
+				layout.Rigid(func(gtx C) D {
+					leftTextLabel := pg.theme.Body1(leftText)
```
This text size should be 14SP. Body1 label is 16SP
